lego/apps/companies/models.py

```diff
@@ -57,6 +57,7 @@ class Company(BasisModel):
 
     class Meta:
         permission_handler = CompanyPermissionHandler()
+        ordering = ["name"]
```

Comment:
Why is this needed?

Reply:
It's the default value. Without it, the front end crashes when accessing bdb without providing ordering in the query params